### ISSUE TITLE
How to prepare stack-exchange data for further pre-training bert

### ISSUE THREAD

**Issue by iamxpy** [oleg-yaroshevskiy/quest_qa_labeling#1]
I notice that SE data can be downloaded from [here](https://archive.org/details/stackexchange), but still I cannot find where to download the files used in [stackexchange-pretrain-large.py](https://github.com/iamxpy/quest_qa_labeling/blob/b2da2b17d995c6939720bd36daf0b5d3bbbafaf0/monty/stackexchange-pretrain-large.py#L42). 

Am I missing something? Or could you provide more details? Thanks in advance!

**Comment by oleg-yaroshevskiy** [comment]
Hi,
thank you for a nice question. I referred that to @mponty . He is going to write a tutorial on how to pretrain LM on masked words and auxiliary targets (`'question_score', 'question_view_count', 'question_favorite_count', 'answer_score', 'answers_count', 'is_answer_accepted'` in this case). 

I believe you can engineer these features yourself from a raw StackExchange dump.

**Comment by Yorko** [comment]
All data preparation and LM fine-tuning code is now [here](https://github.com/oleg-yaroshevskiy/quest_qa_labeling/tree/master/step1_lm_finetuning)
